frontend/coprs_frontend/coprs/views/backend_ns/backend_general.py

```diff
@@ -191,7 +190,7 @@ def get_build_record(task, for_backend=False):
 
     except Exception as err:
         app.logger.exception(err)
-        return None
+        raise err
```

Comment:
This is weird.  app.logger.exception dumps the traceback into logs for the first time, and if we re-raise - some other exception handler will log it once more?